vault/expiration.go

```diff
@@ -429,6 +436,9 @@ func (m *ExpirationManager) revokeCommon(leaseID string, force, skipToken bool)
 func (m *ExpirationManager) RevokeForce(prefix string) error {
 	defer metrics.MeasureSince([]string{"expire", "revoke-force"}, time.Now())
 
+	m.restoreLock()
```

Comment:
Since the first thing in revokeCOmmon is to call loadEntry, wouldn't it be sufficient to put the lock acquisition in loadEntry?

Reply:
Some things load and do additional processing.  We want to hold the lock for the entirety of the call.  Some of the "common" methods have pre and post processing that is done so we just take the lock on all the top level calls.